dsn.go

```diff
@@ -42,22 +42,6 @@ type Config struct {
 	location             *time.Location
 	schema               string
 	transactionIsolation uint32
-
-	user     string
-	password string
-
-	authentication      authentication
```

Comment:
These should not be removed, because for most use-cases, people should just use the DSN to set up what they need instead of having to use the connector and call the HTTP wrapping methods.

Reply:
done